### PR TITLE
DIV-6233: Bailiff service application received

### PR DESCRIPTION
**Pull request by HarryH96**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DIV-6233

### Change description ###
- Adding Bailiff to Service Application Received Screen
- NOTE: The Fixed List "ServiceApplicationPaymentTypeEnum" is behind a deemed&dispensed toggle

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: definitions/divorce/json/FixedLists/FixedLists.json [hunk 1]
```diff
@@ -1238,6 +1238,12 @@
     "ListElementCode": "dispensed",
     "ListElement": "Dispensed with service"
   },
+  {
+    "LiveFrom": "23/07/2020",
+    "ID": "ServiceApplicationTypeEnum",
+    "ListElementCode": "bailiff",
+    "ListElement": "Bailiff Application"
```
Comment (shaunganley) [inline definitions/divorce/json/FixedLists/FixedLists.json:1245]:
```suggestion
    "ListElement": "Bailiff application"
```

**Review by shaunganley — COMMENTED** [review]
how is this toggled off? is the event toggled therefore this option will also be toggled?

**Comment by HarryH96** [comment]
@shaunganley it is contained within the - CaseField-deemed-and-dispensed-nonprod.json

**Review by ocarleton — APPROVED** [review]

**Review by shaunganley — CHANGES_REQUESTED** [review]
as discussed, we'll need to put this behind a toggle

**Review by shaunganley — APPROVED** [review]

**Pull request merged**